cmds/rm/rm.go

```diff
@@ -37,27 +34,20 @@ func init() {
 		os.Args[0] = cmd
 		defUsage()
 	}
-	flag.BoolVar(&flags.i, "i", false, "Interactive mode.")
-	flag.BoolVar(&flags.v, "v", false, "Verbose mode.")
-	flag.BoolVar(&flags.r, "R", false, "Remove file hierarchies")
-	flag.BoolVar(&flags.r, "r", false, "Equivalent to -R.")
-	flag.Parse()
 }
 
 func rm(files []string) error {
```

Comment:
This function should take recursive / interactive as an argument, or it shouldn't be a function. It's not generally nice to access globals outside of main.